src/loader.js

```diff
@@ -98,14 +98,15 @@ function collectStyles(src, filename, resolveDependency, opts) {
 
   // quick regex as an optimization to avoid parsing each file
   if (
-    !src.match(
-      new RegExp(
-        `(${tagName}|${styledTag}(.|\\n|\\r)+?)\\s*\`([\\s\\S]*?)\``,
-        'gmi',
-      ),
-    ) &&
-    opts.cssPropEnabled &&
```

Comment:
this was always false, so it turns out we were never hitting the code below.

we could also just scrap this check altogether